tests/e2e/specs/shopper/shopper-subscriptions-purchase-multiple-subscriptions.spec.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { merchant, shopper, withRestApi } = require( '@woocommerce/e2e-utils' );
+
+import {
+	RUN_SUBSCRIPTIONS_TESTS,
+	describeif,
+	merchantWCP,
+	shopperWCP,
+} from '../../utils';
+
+import { fillCardDetails, setupCheckout } from '../../utils/payments';
+
+const productNames = [
+	`Multi sub 1 ${ Date.now() }`,
+	`Multi sub 2 ${ Date.now() }`,
+];
+const customerBilling = config.get( 'addresses.customer.billing' );
+const card = config.get( 'cards.basic' );
+const productIds = [];
+
+describeif( RUN_SUBSCRIPTIONS_TESTS )(
+	'Subscriptions > Purchase multiple subscriptions',
+	() => {
+		// Setup 2 subscription products.
+		// Remember their product id's.
+		beforeAll( async () => {
+			await merchant.login();
+
+			for ( const productName of productNames ) {
+				const productId = await merchantWCP.createSubscriptionProduct(
+					productName
+				);
+				productIds.push( productId );
+			}
+
+			await merchant.logout();
+		} );
+
+		afterAll( async () => {
+			// Delete the user created with the subscription
+			await withRestApi.deleteCustomerByEmail( customerBilling.email );
```

Comment:
I am not sure if we should delete the products that were created in `beforeAll()`. Should we?
I am afraid that there would be too many of those products after some time.

Reply:
@rodelgc

Reply:
Hi @oaratovskyi thanks for reviewing! 🙂 

Right now, the only way to delete products in WP admin site is by navigating through the UI. But I'm worried that it might add to the already existing flakiness, as we have been getting reports that some e2e tests are getting flaky recently. 

I and @veljkho will be addressing the flakiness of other e2e tests soon. So I'm planning to add optimizations like the one you pointed out here on my next PR.